client/src/common/types/entities.ts

```diff
@@ -1,3 +1,73 @@
 // If a globally shared type or interface doesn't have a clear owner, put it here
 
 export {};
+
+export interface AnnotationColumn {
```

Comment:
this would be better made into a union, so that it can reflect that `categories` is mandatory for some types, optional for others.